packages/manager/modules/pci/src/projects/project/kubernetes/components/proxy-form/proxy-form.styles.scss

```diff
@@ -0,0 +1,31 @@
+.kubernetes-proxy-form {
+  .container-fluid {
+    padding: 0;
+    .col-xs-12 {
+      margin-bottom: 0.5rem;
+      white-space: nowrap;
+    }
```

Comment:
why overriding css bootstrap intead of adding a new one additionally ?

Reply:
This is an addition, not an override. `col-xs-*` have no original `margin-bottom` nor `white-space`